packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -196,17 +184,20 @@ class AudioPlayerWithYoutubeSpotify extends Component {
    */
   getTrackToHighlight(audioSource) {
     const { trackSelected, trackStartingPoint } = this.state;
-    if (Number.isInteger(audioSource.index)) {
+    if (parseInt(audioSource.index, 10) >= 0) {
```

Comment:
QUESTION: Significance of magic number 10, here and below?

Reply:
eslint, [radix](https://eslint.org/docs/rules/radix) rule made me put it there. maybe we can turn off rule...